Description: Add code to set client data in wxVListBoxComboPopup::Insert()
Author: JMS
Bug: http://trac.wxwidgets.org/ticket/13244
Applied-Upstream: http://trac.wxwidgets.org/changeset/67783
Last-Update: 2012-05-31

Index: wxWidgets/branches/WX_2_8_BRANCH/src/generic/odcombo.cpp
===================================================================
--- a/src/generic/odcombo.cpp (revision 64259)
+++ b/src/generic/odcombo.cpp (revision 67783)
@@ -498,4 +498,9 @@
 
     m_strings.Insert(item,pos);
+
+    // Only insert client data if client data is used
+    if ( (int)m_clientDatas.size() >= pos )
+        m_clientDatas.Insert(NULL, pos);
+
     m_widths.Insert(-1,pos);
     m_widthsDirty = true;
